core/src/main/resources/swagger/api-docs.yaml

```diff
@@ -5032,6 +5032,9 @@ definitions:
       enableFlowLogs:
         type: boolean
         description: Optional, false if not specified. If true, enable flow logs within the high security network. Requires highSecurityNetwork = true.
+      privateIpGoogleAccess:
```

Comment:
why is it not named `privateGoogleAccess`?  This seems to be how it is referenced in all the documentation.

Reply:
The actual flag that gets set on the Google subnetwork resource is `privateIpGoogleAccess`, so we went with that even though most documentation does reference it as 'Private Google Access'. See docs here https://cloud.google.com/compute/docs/reference/rest/v1/subnetworks#resource:-subnetwork

Reply:
Definitely open to changing it to `privateGoogleAccess` if you think that's more clear though!